src/pretix/plugins/sendmail/forms.py

```diff
@@ -91,7 +84,95 @@ class MailForm(FormPlaceholderMixin, forms.Form):
         help_text=_('Sending an attachment increases the chance of your email not arriving or being sorted into spam folders. We recommend only using PDFs '
                     'of no more than 2 MB in size.'),
         max_size=settings.FILE_UPLOAD_MAX_SIZE_EMAIL_ATTACHMENT
-    )  # TODO i18n
+    )
+
+    def __init__(self, *args, **kwargs):
+        event = self.event = kwargs.pop('event')
+        context_parameters = kwargs.pop('context_parameters')
+        super().__init__(*args, **kwargs)
+        self.fields['subject'] = I18nFormField(
+            label=_('Subject'),
+            widget=I18nTextInput, required=True,
+            locales=event.settings.get('locales'),
+        )
+        self.fields['message'] = I18nFormField(
+            label=_('Message'),
+            widget=I18nTextarea, required=True,
+            locales=event.settings.get('locales'),
+        )
+        self._set_field_placeholders('subject', context_parameters)
+        self._set_field_placeholders('message', context_parameters)
+
+
+class WaitinglistMailForm(BaseMailForm):
+    items = forms.ModelMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple(
+            attrs={'class': 'scrolling-multiple-choice'}
+        ),
+        label=_('Only send to people who are waiting for'),
+        required=True,
+        queryset=Item.objects.none()
+    )
+    subevent = forms.ModelChoiceField(
+        SubEvent.objects.none(),
+        label=_('Only send to customers of'),
```

Comment:
IMHO the wording „customer“ in the context of a waiting list is a bit strange. I do not have a better wording though maybe one can shorten all of the labels to not include anything like „customer“ or „person“ (and change the order so the product-filter is first – that feels more natural to me) like so:

![Bildschirmfoto 2023-01-10 um 10 34 53](https://user-images.githubusercontent.com/276509/211514863-69b6163a-478f-4141-b50f-0f2c915a7216.png)

The sendmail orders page with subevents would need similiar labels, if we decide to go with this approach.

Reply:
Yes, the and/or part is indeed confusing/hard to grasp. :)